src/main.js

```diff
@@ -36,8 +36,18 @@ let _initPlugin = function(){
   })
 }
 
+let _initOnDOMLoaded = function(repetitions) {
+  if (document.readyState == "complete") {
+    setTimeout(_initPlugin, 1000)
+  } else if (repetitions > 0) {
+    setTimeout(_initOnDOMLoaded, 1000, repetitions - 1)
+  } else {
+    _initPlugin()
+  }
+}
+
 let s = document.createElement('script')
 s.src = 'https://api.addressfinder.io/assets/v3/widget.js'
 s.async = 1
-s.onload = _initPlugin
+s.onload = _initOnDOMLoaded(5)
```

Comment:
because the _initOnDOMLoaded(5) function has brackets on the end, it's actually called immediately, rather than when the onload event happens.